### PR TITLE
feat(pagination): Added aria label input for accessibility

### PR DESCRIPTION
**Pull request by thbt**
Before submitting a pull request, please make sure you have at least performed the following:

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated any applicable tests.
 - [x] added/updated any applicable API documentation.
 - [x] added/updated any applicable demos.

### CODE REVIEW

File: src/pagination/pagination-config.ts [hunk 1]
```diff
@@ -15,4 +15,5 @@ export class NgbPaginationConfig {
   pageSize = 10;
   rotate = false;
   size: 'sm' | 'lg';
+  ariaLabel: string = 'Page navigation';
```
Comment (maxokorokov) [inline src/pagination/pagination-config.ts]: no need in `:string` type

**Comment by thbt** [comment]
Thanks for the review!
1. Makes sense, I removed it from all demo except config.
2. It might not change a lot for the screen readers, but we wouldn't be in line with bootstrap markup anymore if we applied the attribute on the host instead of the inner `<nav>`.

**Comment by pkozlowski-opensource** [comment]
> Maybe I'm talking nonsense, but can't we simply stick [attr.aria-label] on the host element by the way, not on the inner <nav>. They won't read it properly?

@maxokorokov has a point and tbh I would go even a step further - if we would to simply forward a proerty binding to an attribute on a host... why do we need to do things at all? I mean, why not show people to simply use `<ngb-pagination aria-label="...">`?

Could we test it with screen readers and check of we need to do anything here at all?

**Comment by pkozlowski-opensource** [comment]
I've just checked with ChromeVox and it seems like sticking `aria-label` on `<ngb-pagination>` has no effect... So we need to introduce a new input. Was just wondering if we can have it as `aria-label` instead of `ariaLabel` (probably not, but worth trying)?

Other thing: this PR doesn't add `<span class="sr-only">(current)</span></a>` for an active item as done in http://v4-alpha.getbootstrap.com/components/pagination/#disabled-and-active-states - should be added as part of the accessibility work for pagination.

**Comment by pkozlowski-opensource** [comment]
@thbt could you please add support for the `<span class="sr-only">(current)</span>` to this PR as commented above? Ref: http://v4-alpha.getbootstrap.com/components/pagination/#disabled-and-active-states

You might also find this article useful: http://www.a11ymatters.com/pattern/pagination/

**Comment by pkozlowski-opensource** [comment]
LGTM. Let's wait for the TravisCI build and if it is green I'm going to merge this one. Thnx!

**Pull request closed without merging**